modules/ROOT/pages/_partials/dataweave-formats-flatfile-config.adoc

```diff
@@ -14,8 +14,7 @@ DataWeave accepts properties that provide instructions for reading input data in
 | Parameter | Type | Default | Description
 | `enforceRequires` | `Boolean` | `false` | Error if required value missing.
   Valid values are `true` or `false`.
-| `missingValues` | `String` | `NULLS` for copybook schema, `SPACES` otherwise | Fill character used to represent missing
-values. To represent missing values in the input data, you can use:
+| `missingValues` | `String` | `nulls` for copybook schema, `spaces` otherwise | Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
 
 * `NONE`: Treat all data as actual values
```

Comment:
The only thing to mention here is that in Reader properties, the values should be **none**, **spaces**, **zeroes** and **nulls**. 
However, in the Writer properties these should be uppercase: **NONE**, **SPACES**, **ZEROES**, **NULLS**.

Reply:
Thx, Cris. We've got new PRs to address that in our Mule branches (v4.2-v4.4):  
DOCS-12040-missingValues-v44-duke 
#2049 opened 1 hour ago by dukesphere

DOCS-12040-missingValues-v42-duke 
#2048 opened 2 hours ago by dukesphere

DOCS-12040-missingValues-v43-duke 
#2047 opened 2 hours ago by dukesphere

Reply:
Will also make sure this fix gets in our DW repo.